api/src/main/java/jakarta/persistence/EntityManagerFactory.java

```diff
@@ -59,16 +59,36 @@
  *     {@link Persistence#createEntityManagerFactory(PersistenceConfiguration)}.
  * </ul>
  *
+ * <p>Usually, there is exactly one {@code EntityManagerFactory} for
+ * each persistence unit:
+ * {@snippet :
+ * // create a factory at startup time
+ * static final EntityManagerFactory emf =
+ *         Persistence.createEntityManagerFactory("orderMgt");
+ * ...
+ * // destroy the factory before shutting down
+ * emf.close();
```

Comment:
It may make sense to add a suggestion to use java.lang.ref.Cleaner (ie `Cleaner.create().register(emf, () -> emf.close());` to close EMF on app exit

Reply:
Oh, that's something I never even knew about. Yeah, sure, why not?

Reply:
@lukasj WDYT of the update?